test/ServerComparison.FunctionalTests/ServerComparison.FunctionalTests.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\..\build\common.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp1.1;net451</TargetFrameworks>
+    <RuntimeIdentifier>win7-x64</RuntimeIdentifier>
```

Comment:
Make this conditional on not being netcoreapp1.1. `Condition=" '$(TargetFramework)' != 'netcoreapp1.1' "`